src/Ae.js

```diff
@@ -84,30 +84,36 @@ export default class Ae {
     rawTxHex: string,
     networkId: string = "ae_mainnet"
   ): Promise<string> {
+    let isNewAppVersion = true;
+    const appConfig = await this.transport.send(CLA, GET_APP_CONFIGURATION, 0x00, 0x00);
+    if (appConfig[1] === 0 && appConfig[2] < 4) {
+      isNewAppVersion = false;
+    }
```

Comment:
Proposing to use more high-level method to detect version and name variable more semantic:
```suggestion
    const [major, minor] = (await this.getAppConfiguration()).version.split(".").map(n => +n);
    const acceptsTransactionLength = !(major === 0 && minor < 4);
```

Reply:
We actually should drop the getAppConfiguration method out of decorateAppAPIMethods to do so.